src/chrome/content/rules/Red_Hat.xml

```diff
@@ -168,10 +160,10 @@
 		<test url="http://et.redhat.com/foo?bar" />
 
 	<rule from="^http://investors\.redhat\.com/"
-		to="https://investor.shareholder.com/" />
+		to="https://investors.redhat.com/" />
```

Comment:
This rule could just be removed completely to have it be caught by the catch-all at the bottom. The domain should also not be listed under "complications" anymore.